common/nixos/network.nix

```diff
@@ -22,7 +22,6 @@
   # tailscale
   services.tailscale.enable = true;
   services.tailscale.useRoutingFeatures = "client";
-  boot.kernel.sysctl."net.ipv4.ip_forward" = 1;
```

Comment:
Oh, and this one was unnecessary, because it's implied by `services.tailscale.useRoutingFeatures = "both";`